kie-dmn-feel/src/main/java/org/kie/dmn/feel/util/Either.java

```diff
@@ -46,6 +46,6 @@ public boolean isRight() {
     
     public <X> X cata(Function<L,X> left, Function<R,X> right) {
         // warning: left.invoke, because of FEEL specs, could return null. The below is the safest way to implement cata over this Either.
-        return isLeft() ? left.apply( this.left.get() ) : right.apply( this.right.get() );
+        return isLeft() ? left.apply( this.left.get() ) : right.apply( this.right.orElse( null ) );
```

Comment:
Also, comment at line 48 is stating `left.invoke` but now I noticed that was affecting `BaseFEELFunction` in commit 0ba42e7 but here it's the JDK's Function so you can correct it back if you want